tests/cli/test_work_queues.py

```diff
@@ -327,10 +327,42 @@ def test_pause_bad_pool_name(
             expected_code=1,
         )
 
+    def test_pause_without_specifying_pool_name_without_confirmation(
+        self,
+        work_queue,
+    ):
+        invoke_and_assert(
+            command=f"work-queue pause {work_queue.name}",
+            expected_code=1,
+        )
+
+    def test_pause_without_specifying_pool_name_with_confirmation(
+        self,
+        work_queue,
+    ):
+        invoke_and_assert(
+            command=f"work-queue pause {work_queue.name}",
+            user_input="Y",
+            expected_code=0,
+        )
+
+    def test_pause_without_specifying_pool_name_with_abort(
+        self,
+        work_queue,
+    ):
+        invoke_and_assert(
+            command=f"work-queue pause {work_queue.name}",
+            user_input="N",
+            expected_code=1,
+            expected_output_contains="Work queue pause aborted!",
+        )
+
 
 class TestResumeWorkQueue:
     def test_resume(self, prefect_client, work_queue):
-        invoke_and_assert(command=f"work-queue pause {work_queue.name}")
+        invoke_and_assert(
+            command=f"work-queue pause {work_queue.name} --pool default-agent-pool"
+        )
```

Comment:
after this it might be a good idea to do:
```python
q.read_queue(prefect_client, work_queue.name)
assert q.is_paused
```

Reply:
Done :D